internal/corazarules/rule_match.go

```diff
@@ -229,6 +231,22 @@ func (mr MatchedRule) ErrorLog() string {
 		}
 	}
 
-	log.WriteString("\n")
 	return log.String()
 }
+
+func writeDisruptiveActionSpecificLog(log *strings.Builder, mr MatchedRule) {
+	switch mr.DisruptiveActionName_ {
```

Comment:
Maybe for another PR but we should migrate this to an enum instead of string for faster matching

Reply:
Could you please take care of that before merge, @M4tteoP?

Reply:
Thanks Rag, I tried to address it. A better way might be propagating the Action enum from the parser mapping into Action functions, but it requires quite a broad refactor. Did you have in mind something like what I just implemented?